### DISCUSSION TITLE
馒头站点测试无法通过

### DISCUSSION THREAD

**Discussion by huning472847599** [jxxghp/MoviePilot#1996] [问题咨询]
已经更新为最新的 1.85-1，馒头站点测试无法通过，显示为：连通性测试失败：鉴权已过期或无效
如果更新用户名，用户密码 ，两步验证， 则显示：更新失败 未知错误
查看日志：网页操作失败: ENOENT: no such file or directory, mkdtemp '/var/packages/MoviePilot/target/tmp/playwright-artifacts-5GEdzz'
playeright-artiufacts后面都是随机字符， 请大佬看看怎么解决。

**Comment by cng10001** [comment]
有没有添加站点User-Agent试试

**Comment by huning472847599** [comment]
已经测试通过了，
1)之前的COOKIE不能留
2)User-Agent CHROME和EDGE也要分清
感谢
________________________________
发件人: cng10001 ***@***.***>
发送时间: 2024年5月13日 20:03
收件人: jxxghp/MoviePilot ***@***.***>
抄送: huning472847599 ***@***.***>; Author ***@***.***>
主题: Re: [jxxghp/MoviePilot] 馒头站点测试无法通过 (Discussion #1996)


有没有添加站点User-Agent试试

―
